src/vectorchord/advanced-features/prefetch.md

```diff
@@ -0,0 +1,31 @@
+# Prefetch <badge type="tip" text="since v0.4.0" />
+
+The default prefetch setting dramatically accelerates queries on large datasets by loading the necessary data from the disk into the memory cache before it is needed. It minimizes disk I/O latency during queries, allowing users to enjoy optimal performance with no configuration necessary.
+
+## Prefetch Mode
+
+:::tip
+This feature is enabled by default. Any query can be benefit from prefetching with VectorChord `v0.4.0` and later versions.
+:::
+
+PostgreSQL 17 has introduced a new [streaming I/O API](https://www.postgresql.org/docs/current/release-17.html) for more efficient sequential I/O. Take advantage of it, a sequential scan on a table will be much faster.
+
+In VectorChord, we implemented two prefetch methods: A simple one, `prefetch_buffer`, and `read_stream`, which uses the streaming I/O API. Both methods are faster than the previous behavior in most cases, `read_buffer`.
```

Comment:
1. The last sentence contains only the subject.
2. It claims there are two prefetch methods, and then it list three.
3. `prefetch_buffer` is not introduced here either.